.circleci/config.yml

```diff
@@ -367,14 +355,6 @@ workflows:
           context:
             - circleci-slack
           <<: *slack-fail-post-step
-      - admin-tests:
-          filters:
-            branches:
-              ignore:
-                - /update-builder-.*/
```

Comment:
This isn't carried over into the GitHub Actions workflow, but it was obsoleted by #6706 anyway.